site/app/libraries/Utils.php

```diff
@@ -308,11 +308,17 @@ public static function returnBytes(string $size_str): int {
     * E.g : MB, 10485760 => 10MB
     * @param string $format
     * @param int $bytes
+    * @param bool $round should the result be rounded to the nearest number
     * @return string
     */
-    public static function formatBytes(string $format, int $bytes): string {
+    public static function formatBytes(string $format, int $bytes, bool $round = false): string {
         $formats = ['b' => 0, 'kb' => 1, 'mb' => 2];
-        return ($bytes / pow(1024, floor($formats[strtolower($format)]))) . (strtoupper($format));
+        $result = $bytes / pow(1024, floor($formats[strtolower($format)]));
+        if (round($result)) {
```

Comment:
shouldn't this be `if ($round)`?

Reply:
yes whoops - some hackathon code :sweat_smile: